hbase-zookeeper/src/main/java/org/apache/hadoop/hbase/zookeeper/HQuorumPeer.java

```diff
@@ -88,6 +89,15 @@ public static void main(String[] args) {
 
   private static void runZKServer(QuorumPeerConfig zkConfig)
           throws IOException, AdminServer.AdminServerException {
+
+    // Start and schedule the the purge task
+    DatadirCleanupManager purgeMgr=new DatadirCleanupManager(
```

Comment:
nit: let's mention about `purgeInterval` being 0 by default in comment above this and also the fact that providing this config is recommended only for standalone server.